telegram/__init__.py

```diff
@@ -111,8 +119,7 @@ def __init__(self, sh, *args, **kwargs):
             # catch Unauthorized errors due to an invalid token
             self.logger.error("Unable to start up Telegram conversation. Maybe an invalid token? {}".format(e))
         else:
-            if self.logger.isEnabledFor(logging.DEBUG):
-                self.logger.debug("adding command handlers to dispatcher")
+            self.logger.debug("adding command handlers to dispatcher")
```

Comment:
das is enabled for dient zum Zeit sparen weil dann die zeitintensive Aufbereitung der Log Messages mit format gar nicht stattfindet

Reply:
done